### DISCUSSION TITLE
My deployed project is not showing in Vercel Dashboard

### DISCUSSION THREAD

**Discussion by yug-delemont** [vercel/community#6731] [Help] [answered]
### Summary

![image](https://github.com/vercel/community/assets/154723133/26c33ca5-709b-4dfd-8207-ed9d1dc41528)

In my vercel account I was deploy three nextjs projects but it not showing. show only 404.
Can anybody told me what's wrong here?

### Steps to Reproduce

![image](https://github.com/vercel/community/assets/154723133/26c33ca5-709b-4dfd-8207-ed9d1dc41528)

**Comment by robarnow** [comment, marked as answer]
I'm getting the same error on team "designedtorun" - you can actually see most of the account by clicking "Account Settings > Teams > then Manage on your Team, but the Overview page throws a 404. This is a major issue, as we can't reach our deployments.

**Reply by yug-delemont** [reply]
First all of thanks for your reply.
Can we show our deployments in the premium plan?
